cmd/otelcol/config/collector/config.d.linux/receivers/istio.discovery.yaml

```diff
@@ -0,0 +1,110 @@
+#####################################################################################
+# This file is generated by the Splunk Distribution of the OpenTelemetry Collector. #
+#                                                                                   #
+# It reflects the default configuration bundled in the Collector executable for use #
+# in discovery mode (--discovery) and is provided for reference or customization.   #
+# Please note that any changes made to this file will need to be reconciled during  #
+# upgrades of the Collector.                                                        #
+#####################################################################################
+# prometheus/istio:
+#   enabled: true
```

Comment:
Should we release it as disabled initially before the default dashboard are built?

Reply:
yeah, good idea. I'll update the PR